src/main/java/jakarta/inject/package-info.java

```diff
@@ -146,7 +146,7 @@
  * or even plain Java code. An injector could rely on reflection or code
  * generation. An injector that uses compile-time code generation may not even
  * have its own run time representation. Other injectors may not be able to
- * generate code at all, neither at compile nor run time. A "container", for
+ * generate code at all, neither at compilation nor runtime. A "container", for
```

Comment:
This is unnecessary. "At compile time" means "during compilation". "Run time" is probably less common than "runtime", but they do mean the same thing (plus they have other meanings).

Reply:
Yes I will remove this also